themes/dark/templates/editShow.mako

```diff
@@ -12,24 +13,142 @@
+                    const data = {
+                        config: {
+                            aliases: this.series.config.aliases,
+                            dvdOrder: this.series.config.dvdOrder,
+                            flattenFolders: this.series.config.flattenFolders,
+                            anime: this.series.config.anime,
+                            scene: this.series.config.scene,
+                            sports: this.series.config.sports,
+                            paused: this.series.config.paused,
+                            location: this.series.config.location,
+                            airByDate: this.series.config.airByDate,
+                            subtitlesEnabled: this.series.config.subtitlesEnabled,
+                            release: {
+                                requiredWords: this.series.config.release.requiredWords,
+                                ignoredWords: this.series.config.release.ignoredWords,
+                                blacklist: this.series.config.release.blacklist,
+                                whitelist: this.series.config.release.whitelist
+                            }
+                        }
+                    };
+                    try {
+                        const response = await api.patch('series/' + this.seriesSlug, data);
+                        this.saveStatus = 'Successfully patched series';
+                    } catch (error) {
+                        this.saveStatus = 'Problem trying to save series with error' + error + ', sending data: ' + data;
+                    }
+                };
+            },
+            onChangeIgnoredWords: function(items) {
+		        console.debug('Event from child component emitted', items);
+                this.series.config.release.ignoredWords = items.map(item => item.value);
+            },
+            onChangeRequiredWords: function(items) {
+		        console.debug('Event from child component emitted', items);
+                this.series.config.release.requiredWords = items.map(item => item.value);
+            },
+            onChangeAliases: function(items) {
+		        console.debug('Event from child component emitted', items);
+                this.series.config.aliases = items.map(item => item.value);
+            },
+            onChangeReleaseGroupsAnime: function(items) {
+                this.series.config.release.whitelist = items.filter(item => item.memberOf === 'whitelist');
+                this.series.config.release.blacklist = items.filter(item => item.memberOf === 'blacklist');
+                this.series.config.release.allgroups = items.filter(item => item.memberOf === 'releasegroups');
+
+            },
+            saveLocation: function(value) {
+                this.series.config.location = value;
+            }
+        },
+        computed: {
+            availableLanguages: function() {
+                if (this.config.indexers.config.main.validLanguages) {
```

Comment:
@p0psicles is this actually set anywhere in the UI?

Reply:
No, it's read only. It's a stupid thing actually. As it was once added purely for tvdb. Others don't use it